specificatie/openapi.yaml

```diff
@@ -1294,6 +1554,19 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/PandHalBasis'
+    PandHalPaginationCollectie:
+      type: object
+      description: "Resultaten als lijst."
+      properties:
+        _links:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/schemas/HalPaginationLinksMetLast'
+        _embedded:
+          type: object
+          properties:
+            zoekresultaten:
+              type: array
```

Comment:
Hier worden (in tegenstelling tot het zoeken op adressen) daadwerklijk panden teruggeleverd. Deze property zou volgens mij dan ook "panden"moeten zijn en geen "zoekresultaten"